Missing objective data for quest 26628 - Death from Below (Arathi Highlands, Cataclysm Classic)

<!-- READ THIS FIRST

Hello, thanks for taking the time to report a bug!

Before you proceed, please verify that you're running the latest version of Questie. The easiest way to do this is via the Curseforge client, but you can also download the latest version here: https://www.curseforge.com/wow/addons/questie

Questie is one of the most popular Classic WoW addons, with over 30 million downloads. However, like almost all WoW addons, it's built and maintained by a team of volunteers.

If you'd like to help, please consider making a donation. You can do so here: https://www.paypal.com/cgi-bin/webscr?cmd=_donations&business=aero1861%40gmail%2ecom&lc=CA&item_name=Questie%20Devs&currency_code=USD&bn=PP%2dDonationsBF%3abtn_donate_LG%2egif%3aNonHosted

You can also help as a tester, developer or translator. To do so please join the Questie Discord here https://discord.gg/fYcQfv7

-->


## Description
<!-- Explain in detail what problem you encountered. Is a quest not showing or showing again or an objective not showing or a spawn missing? -->

Hey folks! While continuing my adventures through Arathi Highlands, I came across the following error

[ERROR] Missing objective data for quest 26628 Defend Shakes O'Breen

## Wowhead link
<!-- If this issue is about a quest/npc/item/... then please include a link to it from Wowhead: https://wowhead.com/classic or https://wowhead.com/wotlk or https://wowhead.com/cata -->

https://www.wowhead.com/cata/quest=26628/death-from-below

## Screenshots
<!-- If you can, add a screenshot to help explaining the problem (e.g. of your map where a quest is not shown but the NPC has an ! above him). Simply drag and drop the image in this input field, no need to upload it to any other image platform. -->

Questie Error:
![image](https://github.com/user-attachments/assets/0ecc9d35-2996-44fc-b49f-459706bea65f)

Quest Description:
![image](https://github.com/user-attachments/assets/ee50db1a-7c6d-4501-a82c-18e870c14209)

## Questie & WoW version
<!--
Which version of Questie are you using? You can find it by:
- 1. Hovering over the Questie Minimap Icon
- 2. looking at your Questie.toc file (open it with any text editor).
It looks something like this: "v5.9.0" or "## Version: 5.9.0".

Please also state if your quest issue is about Classic Era/SoM or Wotlk.
-->

Questie 10.3.3
WoW 4.4.0 (Cataclysm Classic)

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -4231,6 +4231,9 @@ function CataQuestFixes.Load()
         [26627] = { -- The Hermit
             [questKeys.exclusiveTo] = {26653},
         },
+        [26628] = { -- Death from Below
+            [questKeys.objectivesText] = {"Protect Shakes O'Breen during the attack."},
+        },
         [26629] = { -- Seeing Where Your Loyalties Lie
             [questKeys.childQuests] = {26630},
         },

--- a/Database/Corrections/cataQuestFixes.lua
+++ b/Database/Corrections/cataQuestFixes.lua
@@ -4232,7 +4232,7 @@ function CataQuestFixes.Load()
             [questKeys.exclusiveTo] = {26653},
         },
         [26628] = { -- Death from Below
-            [questKeys.objectivesText] = {"Protect Shakes O'Breen during the attack."},
+            [questKeys.triggerEnd] = {"Defend Shakes O'Breen", {[zoneIDs.ARATHI_HIGHLANDS]={{32.20,81.40}}}},
         },
         [26629] = { -- Seeing Where Your Loyalties Lie
             [questKeys.childQuests] = {26630},

